ql/src/java/org/apache/hadoop/hive/ql/plan/LoadTableDesc.java

```diff
@@ -157,6 +159,19 @@ public LoadTableDesc(final Path sourcePath,
     }
   }
 
+  public LoadTableDesc(Path path, Table tableHandle, boolean isOverWrite, boolean isIcebergLoad) {
+    super(path, AcidUtils.Operation.NOT_ACID);
+    this.mdTable = tableHandle;
+    this.isIcebergLoad = isIcebergLoad;
+    this.loadFileType = isOverWrite ? LoadFileType.REPLACE_ALL : LoadFileType.KEEP_EXISTING;
+    this.table = Utilities.getTableDesc(tableHandle);
+  }
+
+  @Explain(displayName = "Iceberg Table Load")
```

Comment:
http://ci.hive.apache.org/blue/organizations/jenkins/hive-precommit/detail/PR-4392/1/tests/ Qtests failed with the added displayName ` Iceberg Table Load: false`. IMHO, we no need to display the field status, and i think maybe no one care what it is.